src/graphql/error-map.ts

```diff
@@ -80,6 +80,10 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
       message = "Invoice is missing its 'payment secret' value"
       return new InvoiceDecodeError({ message, logger: baseLogger })
 
+    case "CouldNotFindLnPaymentFromHashError":
+      message = `Ln Payment does not exist for payment hash ${error.message}`
```

Comment:
should not be sent to client if we make this error not flaw back to the client because it's not critical?

or is it required because of the Typescript rules?

Reply:
We need to put something for the typescript switch, but it currently never gets to the user because we no longer return that error.